.ci_support/linux_aarch64_python3.8.____cpython.yaml

```diff
@@ -0,0 +1,24 @@
+BUILD:
```

Comment:
We shouldn't be creating this file manually. If conda-smithy is not generating it, there is a reason for that. Please see my other comment for details

Reply:
Understood, I wasn't aware of it John. The readme also says to change inputs in `conda_build_config.yaml`, but couldn't find it.

Reply:
No worries

We can create `conda_build_config.yaml` in the recipe when it is needed, but that shouldn't be needed in this case

Reply:
Also submitted PR ( https://github.com/conda-forge/conda-smithy/pull/1574 ) to refresh the README in `.ci_support`

Reply:
Shouldn't be this file removed, and let the conda-smithy recreate it? Or it doesn't really matter who created it?

Reply:
I think the rerender command took care of re-modifying/re-creating it.